owasp-top10-2017-apps/a5/ecommerce-api/README.md

```diff
@@ -49,7 +49,7 @@ Hey, rafael! This is your ticket: rafael-08e8805f-422b-477b-8565-b0876b89da17
 
 ## Attack Narrative
 
-To understand how this vulnerability can be exploited, check [this] section!
+To understand how this vulnerability can be exploited, check [this section]!
```

Comment:
Suggestion: What do you think of `[this section](docs/ATTACK.md)`?

Reply:
Thanks @rafaveira3 ! But the link to `[this section]` is at the end of the file!